attributes/default.rb

```diff
@@ -190,6 +186,22 @@
                                                 httpd boost-devel redhat-lsb mlocate mpich-devel R atlas-devel fftw-devel
                                                 libffi-devel dkms mysql-devel libedit-devel postgresql-devel postgresql-server
                                                 sendmail cmake byacc libglvnd-devel mdadm libgcrypt-devel]
+    if node['platform_version'].to_i == 2
+      # Swap out some packages for their alinux2 equivalents
+      [%w[db4-devel libdb-devel], %w[mpich-devel openmpi-devel], %w[redhat-lsb system-lsb]].each do |al1, al2equiv|
```

Comment:
let's sync on the swap [mpich-devel openmpi-devel]

Reply:
I messaged you about this but for anyone that's interested: neither of the two will be installed since we already install two different MPI libraries.